.pipelineharmonizationrc

```diff
@@ -1,16 +1,51 @@
 #!/bin/bash
 # Set these variables and run `source .pipelineharmonizationrc` prior to running PipelineHarmonization scripts
-export PH_SERVER_ROOT="/data3/cnl"  # folder on server containing relevant files
-export LOCAL_ROOT="/Users/xinhui.li/Documents/reproducibility"  # folder on local machine containing relevant files
-export BASEFOLDER="${LOCAL_ROOT}/LA/Reproducibility_Analysis"
+# You can change any of these variables to adjust where generated files are stored
+
+
+## Files on local machine
+
+### folder on local machine containing relevant files
+export LOCAL_ROOT="${HOME}/Documents/reproducibility"
+
+#### no need to set these
+#### these are populated based on the variable above
+#### (unless you want different organization)
 export LOCAL_OUTPUT_DIRECTORY="${LOCAL_ROOT}"
-export DATA_INPUT_DIR="${PH_SERVER_ROOT}/xli/reproducibility/out"
-export SCRIPT_DIR="${PH_SERVER_ROOT}/xli/reproducibility/script"
+export BASEFOLDER="${LOCAL_ROOT}/LA/Reproducibility_Analysis"
```

Comment:
The local file organization is a bit messy.. we don't have to follow this file structure :)